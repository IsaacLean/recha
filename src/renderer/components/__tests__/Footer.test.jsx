import Adapter from 'enzyme-adapter-react-16'
import Enzyme, { shallow } from 'enzyme'
import React from 'react'

import Footer from '../Footer'

Enzyme.configure({ adapter: new Adapter() })

describe('Footer', () => {
  it('renders properly', () => {
    const component = shallow(<Footer />)
    expect(component).toMatchSnapshot()
  })
})
