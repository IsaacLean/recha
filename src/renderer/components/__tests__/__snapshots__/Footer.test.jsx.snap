// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders properly 1`] = `
<p>
  Show:
   
  <FilterLink
    filter="all"
  >
    All
  </FilterLink>
   
  <FilterLink
    filter="active"
  >
    Active
  </FilterLink>
   
  <FilterLink
    filter="completed"
  >
    Completed
  </FilterLink>
</p>
`;
